iree/compiler/Translation/IREEVM.cpp

```diff
@@ -212,8 +216,10 @@ static LogicalResult translateFromMLIRToVMCModuleWithFlags(
   auto bindingOptions = getBindingOptionsFromFlags();
   auto halTargetOptions = IREE::HAL::getTargetOptionsFromFlags();
   auto vmTargetOptions = IREE::VM::getTargetOptionsFromFlags();
-  return translateFromMLIRToVMCModule(
-      moduleOp, bindingOptions, halTargetOptions, vmTargetOptions, output);
+  auto cCodeTargetOptions = IREE::VM::getCCodeTargetOptionsFromFlags();
+  return translateFromMLIRToVMCModule(moduleOp, bindingOptions,
+                                      halTargetOptions, vmTargetOptions,
+                                      cCodeTargetOptions, output);
```

Comment:
Instead of `cCodeTargetOptions` maybe `emitcTargetOptions` or simpler `cTargetOptions`?